atam4j/src/main/java/me/atam/atam4j/TestRunListener.java

```diff
@@ -10,10 +10,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.lang.reflect.Method;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Totally agree with this.  I strived to find a way to create this test without affecting any code within the project.  The challenge was, how to determine if a test run was in progress?  My first few attempts involved changing the actual project code and exposing more info on the api.  However this felt wrong to me as I was making an arbitrary decision to change the public api based on test convenience.

My final attempt (this one) started life as a branch from the original attempts which is why the production code was modified.  Even though it makes no odds here... I'll revert the change.